modular_skyrat/modules/customization/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -64,11 +68,18 @@
 		if(!uniform_overlay)
 			uniform_overlay = U.build_worn_icon(default_layer = UNIFORM_LAYER, default_icon_file = 'icons/mob/clothing/under/default.dmi', isinhands = FALSE, override_state = target_overlay, override_icon = icon_file, override_x_center = x_override, mutant_styles = applied_style, species = dna.species.species_clothing_path)
 
+		if(U.accessory_overlay)
+			if(STYLE_TESHARI)
+				var/list/teshari_accessory_states = icon_states('modular_skyrat/master_files/icons/mob/clothing/species/teshari/accessories.dmi')
```

Comment:
I feel like we'd be better off with checking if it's in the teshari dmi file once if there's a particular var that's set to false (has_teshari_sprite or something, I dunno, could also be a flag), and then set it to true once we find it, so we don't have to loop through here every single time. Also, static please
```suggestion
				var/static/list/teshari_accessory_states = icon_states('modular_skyrat/master_files/icons/mob/clothing/species/teshari/accessories.dmi')
```